rakelib/form526.rake

```diff
@@ -4,40 +4,159 @@ require 'pp'
 require 'set'
 
 namespace :form526 do
-  desc 'Get all submissions within a date period. [<start date: yyyy-mm-dd>,<end date: yyyy-mm-dd>]'
-  task :submissions, %i[start_date end_date] => [:environment] do |_, args|
+  desc <<~HEREDOC
+    Get all submissions within a date period:
+      rake form526:submissions[2021-01-23,2021-01-24]
+    Last 30 days:
+      rake form526:submissions[]
+    BDD stats mode:
+      rake form526:submissions[bdd]
+  HEREDOC
+  task :submissions, %i[first second] => [:environment] do |_, args|
```

Comment:
Would it make more sense to name the arguments something like first_date, second_date just to make it clear that the arguments should be dates?

Reply:
I could! I was kinda torn though. There is one scenario where the first arg isn't a date (it's just the string "BDD").